[SEO 18.] Add more docs + Vitest (#224)
KnapsackPro/docs.knapsackpro.com: docusaurus/docs/cypress/cookbook.md

| Semaphore 2.0  | `SEMAPHORE_WORKFLOW_ID`  |
| Travis         | `TRAVIS_BUILD_ID`        |
| Codefresh.io   | `CF_BUILD_ID`            |

## Run a subset of tests

> To run a subset of your test suite you have a couple of options:
> 
> - `KNAPSACK_PRO_TEST_FILE_*` environment variables:

Reviewer: ```suggestion
To run a subset of your test suite you can use the `KNAPSACK_PRO_TEST_FILE_*` environment variables:
```

Reviewer: Same for the other JS cookbooks
